xbmc/music/tags/TagLoaderTagLib.cpp

```diff
@@ -1048,10 +1057,27 @@ void CTagLoaderTagLib::SetGenre(CMusicInfoTag &tag, const std::vector<std::strin
 
 void CTagLoaderTagLib::SetReleaseType(CMusicInfoTag &tag, const std::vector<std::string> &values)
 {
+  tag.SetBoxset(false);
   if (values.size() == 1)
+  {
+    if (values[0].find("boxset") != std::string::npos)
+    {
+      tag.SetBoxset(true);
+      if (tag.GetDiscSubtitle().empty()) // no disc title for the tracks on this boxset disc
+        tag.SetDiscSubtitle("dummy");
+    }
     tag.SetMusicBrainzReleaseType(values[0]);
+  }
   else
+  {
     tag.SetMusicBrainzReleaseType(StringUtils::Join(values, CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_musicItemSeparator));
+    if (tag.GetMusicBrainzReleaseType().find("boxset") != std::string::npos)
+    {
```

Comment:
Wondering if all this manipulation of  this tag value to get boxed set flag should happen when saving to the db (so in CMusicDatabase or CSong/CAlbum  rather than here in the tag reading process.